src/libultra/io/contramread.c

```diff
@@ -4,51 +4,53 @@
 
 s32 __osPfsLastChannel = -1;
 
+#define READFORMAT(ptr) ((__OSContRamReadFormat*)(ptr))
```

Comment:
Could this be defined in a common header maybe? It's defined in 3 different files right now which is a bit unfortunate